src/python/pants/backend/helm/subsystems/helm.py

```diff
@@ -44,3 +45,7 @@ def generate_exe(self, plat: Platform) -> str:
     @property
     def lint_strict(self) -> bool:
         return cast("bool", self.options.lint_strict)
+
+
+def rules():
+    return [SubsystemRule(HelmSubsystem)]
```

Comment:
I recall that `SubsystemRule` is not necessary unless the subsystem is never used in a rule. Did any particular error motivate adding it here?

Reply:
not an error really, thought I was missing something as I had to explicitly add them in the tests to the `RuleRunner`. Will remove it.